addons/stock/models/stock_quant.py

```diff
@@ -173,6 +190,22 @@ def write(self, vals):
             return super(StockQuant, self).write(vals)
         return super(StockQuant, self).write(vals)
 
+    def fields_view_get(self, view_id=None, view_type='form', toolbar=False, submenu=False):
+        res = super().fields_view_get(view_id=view_id, view_type=view_type, toolbar=toolbar, submenu=submenu)
+        if res['type'] == 'search' and self.env.context.get('onhand_filter_domain'):
+            doc = etree.XML(res['arch'])
+            magic_filter_attrib = {
```

Comment:
![](https://s2.qwant.com/thumbr/0x380/c/b/c178b1badcaabd85d4acdedd001ad460d48f9926709cbb0e64e4450e205033/MAgic.jpg?u=https%3A%2F%2Fwww.basket4us.com%2Fblog%2Fwp-content%2Fuploads%2F2016%2F06%2FMAgic.jpg&q=0&b=1&p=0&a=1)

Reply:
could you add a comment to explain what you are doing here ?